app/models/megaphone/episode.rb

```diff
@@ -319,13 +340,14 @@ def get_media_info(enclosure)
       raise err
     end
 
-    def arrangement_version_url(location, media_version, count)
+    def arrangement_version_url(location, media_version)
```

Comment:
we don't want the request count in the file name - we really want the media version id to match against